src/explore-education-statistics-common/src/components/__tests__/Button.test.tsx

```diff
@@ -97,46 +78,30 @@ describe('Button', () => {
 
     expect(button).toBeEnabled();
 
-    userEvent.click(button);
+    await userEvent.click(button);
 
     expect(handleClick).toHaveBeenCalledTimes(1);
     expect(button).toBeEnabled();
 
-    jest.advanceTimersByTime(1000);
-    // Flushes promise queue so any state change is triggered
-    await flushPromises();
-
     // Button is still enabled
     expect(button).toBeEnabled();
-
-    jest.runOnlyPendingTimers();
-    jest.useRealTimers();
   });
 
   test('enabled once the current `onClick` handler has finished', async () => {
-    jest.useFakeTimers();
-
-    const handleClick = jest.fn(
-      () => new Promise(resolve => setTimeout(resolve, 2000)),
-    );
+    const handleClick = jest.fn(async () => delay(100));
```

Comment:
Can't help but notice that this method in the file below is `... => await delay(100));` whereas the same method appears here without awaiting the delay. Presumably delay should be always be awaited?

Reply:
That's just a style difference, if you're returning it directly you don't need to await it and will get an eslint error if you do - `Redundant use of `await` on a return value`. I've updated it in `ButtonText.test.tsx` to be like it is here as it's more concise.